### PR TITLE
MAINT: Use positional only to make code a bit more idiomatic.

### PR DESCRIPTION
**Pull request by Carreau**
Up until we were ssupporting 3.7,  we had to pass args/kwargs as non-unpacked arguments in in _make_viewer_then, as otherwise `add_method` could be ambiguous.

Now that we support 3.8 only, we can simplify a bit _make_viewer_then.

- This avoid manually popping `viewer` from kwargs if it exists.
- Avoid making intermediate dict and tuple.

I think it is a bit easier to read, and that *args/**kwargs are a bit better supported by linter/type inference. While we are still highly dynamic here with the method we will call, at least we don't go through opaque Dict[str, any].

# Description

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change
- [ ] example: I check if my changes works with both PySide and PyQt backends
      as there are small differences between the two Qt bindings.  

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/developers/translations.html).

### CODE REVIEW

**Review by brisvag — APPROVED** [review]
I like this change, it feels more pythonic to me.

File: napari/view_layers.py [hunk 1]
```diff
@@ -117,8 +117,10 @@ def _merge_layer_viewer_sigs_docs(func):
 
 def _make_viewer_then(
     add_method: str,
-    args,
-    kwargs,
+    /,
```
Comment (Czaki) [inline napari/view_layers.py:120]: how args will work after `/`?

Comment (brisvag) [reply napari/view_layers.py:120]: This is `/` , which affects what comes before it, unlike `*`. Which affects what comes after!

Comment (Czaki) [reply napari/view_layers.py:120]: ok, i see

Comment (Carreau) [reply napari/view_layers.py:120]: For future reference:

https://peps.python.org/pep-0570/

**Review by goanpeca — APPROVED** [review]
Love it 🚀 !

**Pull request merged**